shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.ivy.ui
+
+import androidx.compose.runtime.Composable
+import com.ivy.domain.features.BoolFeature
+import com.ivy.domain.features.FeatureGroup
+import com.ivy.domain.features.Features
+import com.ivy.ui.time.DevicePreferences
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Test
+import java.util.Locale
+
+class FormatMoneyUseCaseTest {
+
+    private val features = mockk<Features>()
+    private val devicePreferences = mockk<DevicePreferences>()
+    private val showDecimal = BoolFeature(
+        key = "show_decimal_number",
+        group = FeatureGroup.Other,
+        name = "Show Decimal Number",
+        description = "Show Decimal Number in amounts",
+        defaultValue = true
+    )
+
+    private val hideDecimal = BoolFeature(
+        key = "show_decimal_number",
+        group = FeatureGroup.Other,
+        name = "Show Decimal Number",
+        description = "Show Decimal Number in amounts",
+        defaultValue = false
+    )
+
+    private lateinit var formatMoneyUseCase: FormatMoneyUseCase
+
+    @Before
+    fun setup() {
+        formatMoneyUseCase = FormatMoneyUseCase(features,devicePreferences)
+    }
+
+    @Composable
```

Comment:
Interesting! To this test work?

Reply:
` feature.showDecimalNumber.asEnabledState()` is a composable, thus FormatMoneyUseCase.format(value: Double) becomes a composable. Is there any alternate to perform unit test on such function?